shithead_game_server/src/lobby.rs

```diff
@@ -179,6 +210,32 @@ impl Lobby {
             }
         }
     }
+
+    /// Starts the game in this lobby without performing any checks.
+    /// The checks are done in `GameServerState::start_game`.
+    ///
+    /// This function gives all players the initial amount of cards.
+    pub fn start_game(&mut self) {
+        self.state = LobbyState::GameStarted;
+
+        for player in self.players.values_mut() {
+            player.cards_in_hand = self
+                .deck
```

Comment:
why do u write expect here instead of doing it in the `take_cards_from_top`, it's repetitive...

Reply:
Because at some point in the game i will want to take some cards from the top **if i have any left** and otherwise do nothing. I can split it into 2 different functions if u think it makes more sense.